.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,30 @@
+on: 
+  push:
+    tags:
+      - '*'
+workflow_dispatch
+
+jobs: 
+  flatpak:
+     
+    runs-on: ubuntu-20.04
+    
+    steps:
+    - uses: actions/checkout@v2 
+    - name: Install Flatpak
+      run: >
+        sudo apt install flatpak && flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+        flatpak install flathub org.gnome.Platform//3.36 org.gnome.Sdk//3.36 org.flatpak.Builder
```

Comment:
This requires the `-y` flag, to accept installing the right flatpak dependencies (9 of them?)

Reply:
The Platform and Sdk also have Locale dependencies, so that is 4; the other 5 was probably from having flatpak install the builder instead of apt.  The commands have been altered to take out the flatpak builder from flathub, and to use apt instead so there should be 4 dependencies in the flatpak install step now.